src/core/QCheck.mli

```diff
@@ -1254,31 +1302,67 @@ val string : string arbitrary
 val small_string : string arbitrary
 (** Same as {!string} but with a small length (ie {!Gen.small_nat} ). *)
 
+val string_small : string arbitrary
+(** Same as {!string} but with a small length (ie {!Gen.small_nat} ).
+    @since NEXT_RELEASE *)
+
```

Comment:
I propose documenting that this is a convenient alias for `small_string` in the interface, and similarly for the other `string arbitrary` generator aliases. Otherwise it is hard as a new-comer to tell the difference between two similarly named generators with identical documentation :shrug: